mecanum-driveXbox/robot.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+    This is a demo program showing how to use Mecanum control with the
+    MecanumDrive class.
+"""
+
+import wpilib
+from wpilib.drive import MecanumDrive
+#Import the Xbox specific control
+from wpilib.interfaces import GenericHID
+
+Hand = GenericHID.Hand
+
+class MyRobot(wpilib.TimedRobot):
+    # Channels on the roboRIO that the motor controllers are plugged in to
+    frontLeftChannel = 2
+    rearLeftChannel = 3
+    frontRightChannel = 1
+    rearRightChannel = 0
+
+    # The channel on the driver station that the joystick is connected to
+    joystickChannel = 0
+    
+
+    def robotInit(self):
+        """Robot initialization function"""
+        self.frontLeftMotor = wpilib.Talon(self.frontLeftChannel)
+        self.rearLeftMotor = wpilib.Talon(self.rearLeftChannel)
+        self.frontRightMotor = wpilib.Talon(self.frontRightChannel)
+        self.rearRightMotor = wpilib.Talon(self.rearRightChannel)
+
+        # invert the left side motors
+        self.frontLeftMotor.setInverted(True)
+
+        # you may need to change or remove this to match your robot
+        self.rearLeftMotor.setInverted(True)
+
+        self.drive = MecanumDrive(
+            self.frontLeftMotor,
+            self.rearLeftMotor,
+            self.frontRightMotor,
+            self.rearRightMotor,
+        )
+
+        self.drive.setExpiration(0.1)
```

Comment:
The default is 0.1, why set it explicitly?

Reply:
I pulled it in from the previous.  I will remove.